app/scripts/directives/blocks.js

```diff
@@ -20,10 +20,14 @@ angular.module('confRegistrationWebApp').directive('nameQuestion', function () {
   return {
     templateUrl: nameQuestionTemplate,
     restrict: 'E',
-    controller: function ($scope) {
+    controller: function ($rootScope, $scope) {
       if (!$scope.answer && !$scope.answer.value) {
         $scope.answer.value = {};
       }
+
+      $scope.lockedStaffProfileBlock =
+        !!$rootScope.globalUser().employeeId &&
```

Comment:
Any chance of `rootScope.globalUser()` returning undefined and throwing an error here?

Reply:
Good catch. That is a possibility if this gets called before the profile finishes loading.